example/CMakeLists.txt

```diff
@@ -36,6 +36,9 @@ add_subdirectory("kernelSpecialization/")
 add_subdirectory("monteCarloIntegration/")
 add_subdirectory("openMPSchedule/")
 add_subdirectory("randomStrategies/")
-add_subdirectory("randomCells2D/")
+# This example is producing invalid memory access during CI tests with the OpenACC backend.
+if(NOT ALPAKA_CI OR NOT ALPAKA_ACC_ANY_BT_OACC_ENABLE)
+    add_subdirectory("randomCells2D/")
+endif()
```

Comment:
This needs to be fixed at some point (discussed offline): #1540.

Reply:
Thanks for opening the issue. It was to late yesterday night to do it.